conda-recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: ioos_qc
-  version: "2.0.0"
+  version: "2.0.0-rc1"
```

Comment:
Since you are building off GitHub you can do:

```yaml
{% set sdata = load_setup_py_data() %}

package:
  name: panel
  version: {{ sdata['version'] }}

source:
  path: ..

build:
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  entry_points:
    {% for group,epoints in sdata.get("entry_points",{}).items() %}
    {% for entry_point in epoints %}
    - {{ entry_point }}
    {% endfor %}
    {% endfor %}

requirements:
  build:
    - python {{ sdata['python_requires'] }}
    {% for dep in sdata['extras_require']['build'] %}
    - {{ dep }}
    {% endfor %}
  run:
    - python {{ sdata['python_requires'] }}
    {% for dep in sdata.get('install_requires',{}) %}
    - {{ dep }}
    {% endfor %}

about:
  home: {{ sdata['url'] }}
  summary: {{ sdata['description'] }}
  license: {{ sdata['license'] }}
```

Reply:
will leave this for a later date, I still build it locally :grimacing: